languages/scala/reference/exercise-concepts/hello-world.md

```diff
@@ -0,0 +1,13 @@
+# Hello World
+
+[Example implementation](https://github.com/exercism/scala/blob/master/exercises/hello-world/example.scala)
+
+# General
+
+- learn how exercism works: submit solution
+- object or case class: used as class wrapper for exercise
+- functions: used as main entry point for the exercise
+- function arguments: input strands passed as arguments
+- return values: return a value from a function
+- make String using `"`
```

Comment:
I think this can be covered under the "strings" Concept (see e.g. https://github.com/exercism/v3/pull/399/files#diff-aef512cf66c0531c60807d4b8c3b5d6fR12).